plugins/modules/aci_l4l7_logical_interface.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_logical_interface
+short_description: Manage L4-L7 Logical Interface (vns:LIf)
+description:
+- Manage Layer 4 to Layer 7 (L4-L7) Logical Interfaces.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  device:
+    description:
+    - The name of an existing Logical Device.
+    type: str
```

Comment:
forgot to add the aliases:
```suggestion
  device:
    description:
    - The name of an existing Logical Device.
    type: str
    aliases: [ logical_device, device_name, logical_device_name ]
```